### DISCUSSION TITLE
Contribute to this project

### DISCUSSION THREAD

**Discussion by virendracarpenter** [DjangoCRM/django-crm#7] [Q&A] [answered]
Hi team,
I'm eager to contribute to this project! I've been following the project's development and I'm impressed by the work so far.
To get started, could you please point me in the right direction for potential contributions? Are there any specific areas or issues that you're currently focusing on? I'm open to taking on tasks of varying complexity.
Thanks for considering!

**Comment by DjangoCRM** [comment, marked as answer]
Hi Virendra.
Welcome aboard!
Many features can be added to this CRM system.
A Contributing file will be added later and new features will be offered.
In the meantime, it is better to focus on solving the problems that arise for users who are just starting to get acquainted with the project.
I suggest you solve the problem you have already encountered (Issue #8). The necessary information will be provided.
Will you try?

**Reply by virendracarpenter** [reply]
Yes we should first focus on fixing bugs. So that project provide a strong foundation.
